.github/workflows/postsubmit.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Postsubmit
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request:  # TODO: DO NOT SUBMIT. remove this
+
+jobs:
+  build_runtime:
+    uses: ./.github/workflows/build_runtime_cmake.yml
+    with:
+      build-dir: build-runtime
+
+  test_runtime:
+    needs: build_runtime
```

Comment:
Names are mixing `underscore_case` and `Sentence case`. Want to tweak a bit?

checks view:
![image](https://user-images.githubusercontent.com/4010439/178787449-bb4d8b90-73f3-4405-9661-f9379cb4a734.png)

actions/jobs view:
![image](https://user-images.githubusercontent.com/4010439/178787554-cba411b8-2fb5-4541-b573-da634218a1c3.png)

Reply:
Yeah I didn't realize that the step calling the workflow would have its name included as well. How much do we actually care about the title case here? I'm not sure it adds much